contracts/evoting/evoting.go

```diff
@@ -771,6 +771,56 @@ func (e evotingCommand) deleteForm(snap store.Snapshot, step execution.Step) err
 	return nil
 }
 
+// manageAdminForm implements commands. It performs the ADD or REMOVE ADMIN command
+func (e evotingCommand) manageAdminForm(snap store.Snapshot, step execution.Step) error {
+	msg, err := e.getTransaction(step.Current)
+	if err != nil {
+		return xerrors.Errorf(errGetTransaction, err)
+	}
+
+	var form types.AdminForm
+	var formID []byte
+
+	txAddAdmin, okAddAdmin := msg.(types.AddAdmin)
+	txRemoveAdmin, okRemoveAdmin := msg.(types.RemoveAdmin)
```

Comment:
so after thinking about this for a bit, I think the problem with doing it like this is that you could perform an `AddAdmin` operation when the frontend sends a `RemoveAdmin` operation and vice-versa, since you're only checking that it is one of these two operations, but not which one

by passing along an argument that tells us what the expected operation is we can avoid this, so is it possible to send `types.AddAdmin` or `types.RemoveAdmin` as argument?